kmodes/kprototypes.py

```diff
@@ -130,13 +130,17 @@ def __init__(self, n_clusters=8, max_iter=100, num_dissim=euclidean_dissim,
                       "Setting n_init to 1.")
             self.n_init = 1
 
-    def fit(self, X, y=None, categorical=None):
+    def fit(self, X, y=None, categorical=None, sample_weights=None):
```

Comment:
I think we can also add it to `KModes`, since you've done most of the legwork for that already.

This can be a follow-up PR.

Reply:
It also seemed to me as if it was nice to enable the functionality for `KModes` as well.

If possible, I would appreciate it if this could be done in a follow-up PR.

Reply:
Sure, that would be great